CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ARQMA_SOURCE_DIR ${ARQMA_DIR}
         CACHE PATH "Path to the root directory for Arqma")
 
 # set location of arqma build tree
-set(ARQMA_BUILD_DIR ${ARQMA_SOURCE_DIR}/build/Linux/release-v0.6.0/release
+set(ARQMA_BUILD_DIR ${ARQMA_SOURCE_DIR}/build/release/
```

Comment:
If you add “/“ at the end output “${ARQMA_SOURCE_DIR}/something” will look:
“~/arqma//something”